vars/kibanaCoverage.groovy

```diff
@@ -1,3 +1,46 @@
+def downloadPrevious(title) {
+  def vaultSecret = 'secret/gce/elastic-bekitzur/service-account/kibana'
+
+  withGcpServiceAccount.fromVaultSecret(vaultSecret, 'value') {
+    kibanaPipeline.bash('''
+
+    gsutil -m cp -r gs://elastic-bekitzur-kibana-coverage-live/previous_pointer/previous.txt . || echo "### Previous Pointer NOT FOUND?"
+
+    if [ -e ./previous.txt ]; then
+      mv previous.txt downloaded_previous.txt
+      echo "### downloaded_previous.txt"
+      cat downloaded_previous.txt
+    fi
+
+    ''', title)
+
+    def previous = sh(script: 'cat downloaded_previous.txt', label: '### Capture Previous Sha', returnStdout: true).trim()
```

Comment:
You can just do `def previous = readFile(file: 'downloaded_previous.txt')`. Shelling out to `cat` works too, but there's a built-in step for this

Reply:
I knew you'd have an insight on this!  Thanks B!